contracts/src/BeefyClient.sol

```diff
@@ -462,6 +461,14 @@ contract BeefyClient {
         nextValidatorSet.length = _nextValidatorSet.length;
         nextValidatorSet.root = _nextValidatorSet.root;
         nextValidatorSet.usageCounters = createUint16Array(nextValidatorSet.length);
+
+        // Clear latest MMR Root if present
+        delete latestMMRRoot;
+    }
+
+    /// @dev Renounce all administrative privileges
+    function renounce() external {
+        delete forceCheckpointOperator;
     }
```

Comment:
Not sure fully understand why we provide an external `renounce` here.

I would assume in the interim period we just make `BeefyClient` inherited from 
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/Ownable.sol for access control and after production and stable for a while we remove the owner control with an upgrade.